packages/foundry/src/cloneProjectTemplate.ts

```diff
@@ -0,0 +1,8 @@
+import degit from "degit";
+import { ensureDir } from "fs-extra";
```

Comment:
I am not sure about adding an additional dependency for such a simple function.

Even if technically adding a new dependency is trivial, most of the time we should restrict ourself from doing so for simple function as this one.

Adding this dependency also add a dependency on 3 other dependencies.